pom.xml

```diff
@@ -198,6 +198,27 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <!-- To support SSO login for Greengrass Installer -->
```

Comment:
I don't think we really want a new dependency unless there's a strong justification. We have many ways for customers to provision gg and to get credentials one way or another.
Let's try and help the customer with alternatives first.